types/ejs/test/ejs.cjs.test.ts

```diff
@@ -72,3 +72,9 @@ ejs.cache = LRU(100);
 /** @see https://github.com/mde/ejs/tree/v2.5.7#custom-delimiters */
 ejs.delimiter = '%';
 delete ejs.delimiter;
+
+// https://github.com/mde/ejs#options
+const renderOptions: ejs.Options = {
+    beautify: true,
+    filename: fileName,
+};
```

Comment:
This also needs to use tabs:

```suggestion
const renderOptions: ejs.Options = {
	beautify: true,
	filename: fileName,
};
```